### DISCUSSION TITLE
Adding PlantUML to brew services

### DISCUSSION THREAD

**Discussion by medecau** [Homebrew/discussions#4837] [Writing Formulae/Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.15
ORIGIN: https://github.com/Homebrew/brew
HEAD: e57a87cfe9c15f192cd4fc6695a2b3b7acf0ddf7
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

PlantUML has a server option that is very useful when generating many diagrams, or working on one with frequent updates, think: updating a diagram on vscode.

It would be extra nice if one could `brew services start plantuml` to have it always running in the background.

The server can be started with
`plantuml -picoweb`

More info: https://plantuml.com/picoweb

I am not familiar with ruby, brew formula files, or the brew PR process, and I'm afraid of making a fool of myself by attempting this.

pinging @fxcoudert. You're the only person that shows up on the blame for the plantuml formula.

**Comment by SMillerDev** [comment]
> I am not familiar with ruby, brew formula files, or the brew PR process, and I'm afraid of making a fool of myself by attempting this.

I don't think anyone who wants to contribute to open source projects is a fool. https://docs.brew.sh/Updating-Software-in-Homebrew can help you with the basic process, and for the syntax you can just check any existing `service do` block or the documentation at https://docs.brew.sh/Formula-Cookbook#service-files. And when you're stuck, just make a draft PR and tag me. It's much easier to do this than you might think.
